crates/forge-runner/src/running.rs

```diff
@@ -222,7 +226,7 @@ pub fn run_test_case(
     let (entry_code, builtins) = runner
         .create_entry_code(func, &runner_args, initial_gas)
         .unwrap();
-    let footer = runner.create_code_footer();
+    let footer = SierraCasmRunner::create_code_footer();
```

Comment:
Did we change the code of `create_code_footer`?

Reply:
Nope, we just use the linter (blockifier team don't 😅)